apypie/api.py

```diff
@@ -156,22 +156,20 @@ def call(self, resource_name, action_name, params={}, headers={}, options={}):
         if not options.get('skip_validation', False):
             action.validate(params)
 
-        return self._call_action(action, params, headers, options)
+        return self._call_action(action, params, headers, options, **kwargs)
 
-    def _call_action(self, action, params={}, headers={}, options={}):
+    def _call_action(self, action, params={}, headers={}, options={}, **kwargs):
```

Comment:
On a side note: `options` is unused

Reply:
I noticed that too, yeah, but didn't want to creep the PR.